### PR TITLE
FEAT: prepare RC 3.3.2

### PR DESCRIPTION
**Pull request by maxulysse**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1221,7 +1223,7 @@ Initial release of `nf-core/sarek`, created with the [nf-core](http://nf-co.re/)
 
 ### Fixed
 
-- [#665](https://github.com/SciLifeLab/Sarek/pull/665) - Input bam file now has always the same name (whether it is from a single fastq pair or multiple) in the `MarkDuplicates` process, so metrics too
+- [#665](https://github.com/SciLifeLab/Sarek/pull/665) - Input bam file now has always the same spelling of name (whether it is from a single fastq pair or multiple) in the `MarkDuplicates` process, so metrics too
```
Comment (FriederikeHanssen) [inline CHANGELOG.md:1226]: 😱

**Review by FriederikeHanssen — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (maxulysse) [reply CHANGELOG.md:1226]: Where is this coming from?

Comment (maxulysse) [reply CHANGELOG.md:1226]: ooooh

Comment (maxulysse) [reply CHANGELOG.md:1226]: bad copy paste over multiple lines

**Pull request merged**